docs/_news/news12.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: news
+title: New for Codewind 0.12.0
+description: New for Codewind 0.12.0
+keywords: news, new, updates, update, version, IntelliJ
+duration: 1 minute
+permalink: news12
+---
+
+## Codewind 0.12.0
+Thursday 14, May 2020
+
+#### ✨ New Features and Highlights for 0.12.0 ✨
+**IntelliJ Tech Preview Updates**
+We're working to bring you a true Codewind experience with IntelliJ and are pleased to present:
+
+- Codewind for IntelliJ [supports the application metrics dashboard and the **Inject metrics** function](https://github.com/eclipse/codewind/issues/2386).
+- You can [open a shell into the running application container](https://github.com/eclipse/codewind/issues/2030)
+- You will be [notified of log changes, even when the log is not currently selected](https://github.com/eclipse/codewind/issues/2269).
+- When a project is opened in a new window, via actions such as reload due to new projoect creation or restart in debug mode (etc), [the project will be pre-selected in the Codewind view](https://github.com/eclipse/codewind/issues/2037).
+
+**Codewind Quick Guides**
+
+Introducing... Quick Guides! Now that we have [this framework in place](https://github.com/eclipse/codewind-docs/pull/553), we hope to make it easier for you to get started. Check out our first two available [guides](https://www.eclipse.org/codewind/guides.html) today!
+
+Need a quick guide to help you with a particular project? Let us know or help write one by contributing in [Github](https://github.com/eclipse/codewind). 
+
+**IDEs**
+##### VS Code and Eclipse
+- [The **Open Container Shell** action is available for Remote/Kubernetes projects](https://github.com/eclipse/codewind/issues/822).
+    - You must have `kubectl` or `oc` installed locally.
+    - The current kubectl context must point to the cluster with the Codewind instance. 
+    - You must be logged in and have permissions on the cluster to run ```kubectl exec``` into the Codewind application pods. 
+
+##### VS Code
+- Codewind projects that are also VS Code workspace folders are [removed from the workspace when the project is deleted](https://github.com/eclipse/codewind/issues/2662).
+
+
+##### Eclipse
+- When adding a template source, you can [use the **Auto Fill from URL** button to fill in the name and description if they are available from the URL](https://github.com/eclipse/codewind/issues/2697). 
+
+#### List of Fixes
+- In Codewind for Eclipse:
+    - A timeout error was shown when cancelling the debug attach job. This is fixed, and it [no longer clears out the launches when the debug job is terminated so you can still view the console output for the launch](https://github.com/eclipse/codewind/issues/2790). 
+    - In the **Add Existing Project** wizard, [if the browse button is used to select a project from the file system, the radio button no longer switches back to selecting a project from the workspace](https://github.com/eclipse/codewind/issues/2704).
```

Comment:
How about, "In the **Add Existing Project** wizard, if you use the **Browse** button to select a project from the file system, the radio button no longer switches back to selecting a project from the workspace"?

I am assuming the name of the button is "Browse," hence the bold/capitalization. If that's not the name, though, we can omit the bold/capitalization.